src/op_analytics/transforms/interop/README.md

```diff
@@ -0,0 +1,72 @@
+interop
+=======
+
+Objective
+---------
+
+Find how many tokens of different bridgeable types are created each day. 
+
+
+Methodology
+-----------
+
+We consider 3 token types:
+
+- SuperchainERC20
```

Comment:
Maybe we should leave it as ERC7802 here given superchainerc20 still needs to find out give the crosschain mint and burn are given to a specific bridge address